emukit/quadrature/methods/warpings.py

```diff
@@ -1,30 +1,34 @@
+import abc
 import numpy as np
 from typing import Optional
 
 
-class Warping:
+class Warping(abc.ABC):
     """The warping for warped Bayesian quadrature."""
 
+    @abc.abstractmethod
     def transform(self, Y: np.ndarray) -> np.ndarray:
         """Transform from base-GP to integrand.
 
         :param Y: Function values of latent function, shape (n_points, 1).
         :return: Transformed values, shape (n_points, 1).
         """
-        raise NotImplemented
+        pass
 
+    @abc.abstractmethod
```

Comment:
Same here, I think it shall be throwing NotImplemented, as before

Reply:
see above.